apps/site/src/components/NavBar/NavBar.tsx

```diff
@@ -17,63 +17,61 @@ export default function NavBar() {
 
 	return (
 		<div className={styles.nav}>
-			<Navbar expand="lg" className={`bg-body-tertiary ${styles.navbar}`}>
+			<Navbar variant="dark" expand="lg" className={`${styles.navbar}`}>
 				<Container fluid>
+					<Link href="/">
+						<div className={styles.logo}>
+							<Image src={ZotHacksLogo.src} alt="Hacks Logo" fill />
+						</div>
+					</Link>
 					<Navbar.Brand />
```

Comment:
Thought: The div with the image for the logo could be a child of the `Navbar.Brand` component instead of the `Link` component.

Reply:
wait so we dont want the logo to be a link?

Reply:
You can keep the logo as a link by moving the div with the image to be a child of the `Navbar.Brand` component by removing the parent `Link` component and setting the `href="/"` and `as={Link}` attributes of `Navbar.Brand`.